### PR TITLE
Doc updates #3 

### PR DESCRIPTION
**Pull request by DamjanOstrelic**
- gLiveView
- ITN stuff
- topologyUpdater

Probably good idea for @Scitz0 and @gufmar to take a look and confirm I think.

### CODE REVIEW

**Review by Scitz0 — CHANGES_REQUESTED** [review]

File: docs/Scripts/gliveview.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-The tool is independent from other files and can run as a standalone utility that can be stopped/started without affecting the status of cardano-node.
+The tool is independent from other files and can run as a standalone utility that can be stopped/started without affecting the status of `cardano-node`.
 
 ##### Download
 
-If you've used [prereqs.sh](basics.md#pre-requisites), you can skip this part , as this is already set up for you. The tool rely on the common env configuration file.  
-To get current epoch blocks, [cntoolsBlockCollector.sh](Scripts/cntools-blocks.md) script is needed. This is optional and **Guild LiveView** will function without it.
+If you've used [prereqs.sh](basics.md#pre-requisites), you can skip this part, as this is already set up for you. The tool relies on the common `env` configuration file.  
+To get current epoch blocks, the [cntoolsBlockCollector.sh](Scripts/logmonitor.md) script is needed. This is optional and **Guild LiveView** will function without it.
```
[the remark this answers is not in the document]
Comment (DamjanOstrelic) [reply docs/Scripts/gliveview.md:10]: Yep, good catch, fixed the link but not the title :)

File: docs/Scripts/gliveview.md [hunk 2]
```diff
@@ -70,4 +74,5 @@ LEGACY_MODE=false                          # (true|false) If enabled unicode box
 RETRIES=3                                  # How many attempts to connect to running Cardano node before erroring out and quitting
 THEME="dark"                               # dark  = suited for terminals with a dark background
                                            # light = suited for terminals with a bright background
+NO_INTERNET_MODE="N"                       # To skip checking for auto updates or make outgoing connections to guild-operators github repository
```
[the remark this answers is not in the document]
Comment (DamjanOstrelic) [reply docs/Scripts/gliveview.md]: I pushed some new changes now, some questions:

1) Does `logMonitor.sh` have user variables?
2) Separate page for `cnode.sh`? Right now, I just added info on the `node-cli.md`
3) What about `cntools.config`? Separate page? Or should it belong in the CNTools page, maybe somewhere close to where it is first mentioned? Along with CNCLI, this is the only thing with user variables that I still have to check / change.

**Review by gufmar — APPROVED** [review]
LGTM

**Review by Scitz0 — APPROVED** [review]
👍

File: docs/Scripts/gliveview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Scitz0) [reply docs/Scripts/gliveview.md]: > 1. Does `logMonitor.sh` have user variables?

No


> 2. Separate page for `cnode.sh`? Right now, I just added info on the `node-cli.md`

Not sure its needed, its a simple script to launch the node, not sure what to describe on separate page for it. But if you feel you can make a page of it, by all means go for it. 


> 3. What about `cntools.config`? Separate page? Or should it belong in the CNTools page, maybe somewhere close to where it is first mentioned? Along with CNCLI, this is the only thing with user variables that I still have to check / change.

I think it's enough to have a section on CNTools main page like for the other scripts, like cncli, called configuration. But make it clear that its a separate file for this script compared to rest.

[the remark this answers is not in the document]
Comment (DamjanOstrelic) [reply docs/Scripts/gliveview.md]: @Scitz0, I was thinking about point 3) but the CNTools page currently offers good info on the file (not too much detail into its contents but clear about what it's used for and that it is a separate file. I think we can merge this for now as I am still going through the docs updating the existing pages. In the next PRs, I would add a separate page for it with examples on what the variables inside control.

[the remark this answers is not in the document]
Comment (Scitz0) [reply docs/Scripts/gliveview.md]: Ok thats fine with me.

**Review by rdlrt — APPROVED** [review]

**Pull request merged**